src/native/managed/cdacreader/Microsoft.Diagnostics.DataContractReader.Contracts/Data/Array.cs

```diff
@@ -17,8 +17,8 @@ public Array(Target target, TargetPointer address)
 
     public uint NumComponents { get; init; }
 
-    internal static class FieldNames
+    public static class FieldNames
     {
-        internal const string NumComponents = $"m_{nameof(NumComponents)}";
+        public const string NumComponents = $"m_{nameof(NumComponents)}";
```

Comment:
Not part of your change, but if this string is the only reason the Array class needs to be public, it might be nicer to have it be under Constants instead of exposing the Data.Array class.

Reply:
Should strings like this be part of the versioned contract implementation instead?

Reply:
I've moved it into the `Constants` type for now.